torch/_inductor/codecache.py

```diff
@@ -1821,6 +1850,14 @@ def _to_bytes(t: torch.Tensor) -> bytes:
             else:
                 run_command_and_check(link_cmd)
 
+            if use_mmap_weights:
+                with open(output_so, "a+b") as f_so:
+                    so_size = f_so.tell()
+                    # Page align the weights
```

Comment:
Agree with the section idea, but I couldn't figure out how to do it with standard compiler tools (I.e. `as` always adds dummy symbol in the end, and later causes `ld` to crash because `symoff` is still 32-bit value and symbol table is put at the end of the executable)

On the other hand, no harm in padding it to 16K